test/onnx/test_operators.py

```diff
@@ -563,6 +563,15 @@ def test_unsqueeze(self):
         x = torch.randn(3, 4, requires_grad=True)
         self.assertONNX(lambda x: x.unsqueeze(len(x.shape)), x)
 
+    def test_groupnorm(self):
+        x = torch.randn(4, 6, 10, 10, 10, requires_grad=True)
+        self.assertONNX(nn.GroupNorm(3, 6, eps=0.002, affine=False), x)
+
+    def test_groupnorm_noaffine(self):
+        model = torch.nn.GroupNorm(3, 6, 0.4)
+        x = torch.randn(3, 8, 10, 10, requires_grad=True)
+        self.assertONNX(nn.GroupNorm(4, 8, eps=0.002), x)
```

Comment:
why is this test case called 'noaffine', but affine is True? (vs the above other test case where affine=False).